### ISSUE TITLE
Requested Conda osx-arm64 recipe 

### ISSUE THREAD

**Issue by cmastalli** [Gepetto/gepetto-viewer-corba#164]
Hi guys,

It would be nice to have the osx-arm64 recipe for Mac M1 users.
Do you think it is possible to deploy it quickly?

If not, could you provide brief instructions how to compile from source the code?
For instance, how should I install omniORB4 and openscenegraph?

**Comment by traversaro** [comment]
@cmastalli fyi to add a conda-forge package to the macOS arm migration you can just open a PR such as https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2887 . The conda-forge system will add the package (and all its dependencies) to the migration that you can track in https://conda-forge.org/status/#armosxaddition .

**Comment by traversaro** [comment]
> @cmastalli fyi to add a conda-forge package to the macOS arm migration you can just open a PR such as [conda-forge/conda-forge-pinning-feedstock#2887](https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2887) . The conda-forge system will add the package (and all its dependencies) to the migration that you can track in https://conda-forge.org/status/#armosxaddition .

The PR is merged, and now `gepetto-viewer-corba` is part of the migration, see for example from https://raw.githubusercontent.com/regro/cf-graph-countyfair/master/status/armosxaddition.svg?sanitize=true : 
![osxarmmigration](https://user-images.githubusercontent.com/1857049/169147096-9771e5e7-3e5b-47fc-b873-18e5a7cc1454.png)

**Comment by traversaro** [comment]
> How can we request to include a new package in conda-forget? (does someone create the GitHub repository with all recipes? or do we need to do it ourselves?)

In a nutshell, you need to provide a PR to https://github.com/conda-forge/staged-recipes, for example https://github.com/conda-forge/staged-recipes/pull/17935 . Detailed instructions: https://conda-forge.org/docs/maintainer/adding_pkgs.html .

> What happens if there is a package dependency or error that doesn't allow us to deploy a specific recipe?

Every change in a package is a PR (version bump, update in pinned dependencies, ["re-render" of the recipe](https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks) ). If this change result in a CI failure, the PR does not get merged until somebody fixes it. If your recipe is affected by a some other recipe PR not being merged, either you fix the PR in the other recipe, or you find a local workaround in your recipe. 

> How long it takes (in average) to have the migration done?

It depends. If all the recipes are building correctly and the migration PR is quickly merged, probably a few days. Otherwise, it depends on how much time does it take for however is interested in fixing the migration PR and make sure that the CI is green.

**Comment by traversaro** [comment]
An update on this. Fortunatly the migration from qt 5.12 (not available on osx-arm64) to qt/qt-main 5.15 started. For what regards that migration (as described in https://conda-forge.org/status/#qt515 and https://raw.githubusercontent.com/regro/cf-graph-countyfair/master/status/qt515.svg?sanitize=true ) the situation is the following:
![qt_migration](https://user-images.githubusercontent.com/1857049/174395885-83be2391-6b2d-427a-8d5e-4f35fe9ecc1b.png)

So https://github.com/conda-forge/openscenegraph-osgqt-feedstock/pull/5 should unblock it.

**Comment by olivier-stasse** [comment]
Thanks @traversaro for the update !

**Comment by traversaro** [comment]
I guess this was fixed by https://github.com/conda-forge/gepetto-viewer-corba-feedstock/pull/25 ?
